include/inexor/vulkan-renderer/renderer.hpp

```diff
@@ -81,7 +82,7 @@ class VulkanRenderer {
 
     std::vector<VkFramebuffer> frame_buffers;
 
-    std::vector<VkCommandBuffer> command_buffers;
+    std::vector<std::unique_ptr<wrapper::CommandBuffer>> command_buffers;
```

Comment:
I don't think we need a vector of `unique_ptr` here.

Reply:
Ah right. We can control the the point of destruction by clearing the vector.

Reply:
Done.